http/jaxrs-reactive/src/main/java/io/quarkus/ts/http/jaxrs/reactive/MediaTypeResource.java

```diff
@@ -0,0 +1,104 @@
+package io.quarkus.ts.http.jaxrs.reactive;
+
+import static io.quarkus.ts.http.jaxrs.reactive.MediaTypeResource.MEDIA_TYPE_PATH;
```

Comment:
Why is it here?

Reply:
That's one constant that defines path both in resource and tests. Personally I don't care, but I expected Josef to do review and I remember his liking.

Reply:
Provided you agree, I'll leave it there is it hurts nothing and you need to do static import for constants used in class level annotation

Reply:
It's defined on line 22, so I'm wondering why is it imported ...

Reply:
removing it would leave to compilation error

Reply:
ah, `MEDIA_TYPE_PATH` is used on class level

Reply:
just a suggestion, in case you want to remove the static import and mark the annotation like:

```
@Path(MediaTypeResource.MEDIA_TYPE_PATH)
```